src/MSCorEE.Desktop/MSCorEE.cs

```diff
@@ -14,6 +16,11 @@ namespace PInvoke
     public static partial class MSCorEE
     {
         /// <summary>
+        /// The CLSID that may be passed to <see cref="CLRCreateInstance"/> to create an instance of <see cref="ICLRMetaHost"/>.
+        /// </summary>
+        public static readonly Guid CLSID_CLRMetaHost = new Guid("{9280188d-0e8e-4867-b30c-7fa83884e8de}");
```

Comment:
Where did you get this GUID? And is this adequately named or should it include something more precise like the CLR version number?